src/routes/backup/BackupVolume.js

```diff
@@ -0,0 +1,349 @@
+    backupVolumes,
+    v1DataEngineEnabled,
+    v2DataEngineEnabled,
+    onOk(newVolume) {
+      let data = Object.assign(newVolume, { standby: true, frontend: '' })
+      data.size = data.size.replace(/\s/ig, '')
+      dispatch({
+        type: 'backup/createVolume',
+        payload: data,
+      })
+    },
+    onCancel() {
+      dispatch({
+        type: 'backup/hideCreateVolumeStandModalVisible',
+      })
+    },
+  }
+
+  const bulkCreateVolumeStandModalProps = {
+    items: backupVolumesForBulkCreate.map((item) => ({
+      size: item.size,
+      fromBackup: item.lastBackupUrl,
+      volumeName: item.volumeName,
+      backingImage: item.backingImage,
+    })),
+    numberOfReplicas: defaultNumberOfReplicas,
+    visible: bulkCreateVolumeStandModalVisible,
+    nodeTags,
+    diskTags,
+    tagsLoading,
+    backupVolumes,
+    backingImages,
+    v1DataEngineEnabled,
+    v2DataEngineEnabled,
+    onOk(drVolumeConfigs) {
+      dispatch({
+        type: 'backup/bulkCreateVolume',
+        payload: drVolumeConfigs,
+      })
+    },
+    onCancel() {
+      dispatch({
+        type: 'backup/hideBulkCreateVolumeStandModalVisible',
+      })
+    },
+  }
+
+  const workloadDetailModalProps = {
+    visible: workloadDetailModalVisible,
+    item: workloadDetailModalItem,
+    onOk() {
+      dispatch({ type: 'backup/hideWorkloadDetailModal' })
+    },
+    onCancel() {
+      dispatch({ type: 'backup/hideWorkloadDetailModal' })
+    },
+  }
+
+  return (
+    <div className={styles.container}>
+      <Row gutter={24}>
+        <Col lg={18} md={16} sm={24} xs={24}>
+          <BackupBulkActions {...backupBulkActionsProps} />
+        </Col>
+        <Col lg={6} md={8} sm={24} xs={24}>
+          <Filter key={backupFilterKey} {...volumeFilterProps} />
+        </Col>
+      </Row>
+      <BackupVolumeList {...backupVolumesProps} />
+      { restoreBackupModalVisible && currentItem.length === 1 && <RestoreBackupModal key={restoreBackupModalKey} {...restoreBackupModalProps} />}
+      { restoreBackupModalVisible && currentItem.length > 1 && <BulkRestoreBackupModal key={restoreBackupModalKey} {...bulkRestoreBackupModalProps} />}
+      { createVolumeStandModalVisible ? <CreateStandbyVolumeModal key={createVolumeStandModalKey} {...createVolumeStandModalProps} /> : ''}
+      { bulkCreateVolumeStandModalVisible ? <BulkCreateStandbyVolumeModal key={bulkCreateVolumeStandModalKey} {...bulkCreateVolumeStandModalProps} /> : ''}
+      { workloadDetailModalVisible ? <WorkloadDetailModal key={WorkloadDetailModalKey} {...workloadDetailModalProps} /> : ''}
+    </div>
+  )
+}
+
+BackupVolume.propTypes = {
+  backup: PropTypes.object,
+  location: PropTypes.object,
+  dispatch: PropTypes.func,
+  loading: PropTypes.bool,
+  setting: PropTypes.object,
+  backingImage: PropTypes.object,
+}
+
+export default connect(({ backup, setting, backingImage, loading }) => ({ backup, setting, backingImage, loading: loading.models.backup }))(BackupVolume)
```

Comment:
Moved from the `backup/index.js`, no changes to the logic.
Ref: https://github.com/longhorn/longhorn-ui/pull/827/files#diff-e5e7e3c3e107db474c456a6d74c30dbe17e72b2668a86702c0cdfb7ce7b42354L19